### ISSUE TITLE
Training data setup

### ISSUE THREAD

**Issue by sp8va100** [ipeter50/ken-burns-effect#4]
Hi, thanks for your incredible work!

I wanna train the network by myself, but I am confused about how to setup data.
In data_loader.py, it seem that I should download imagenet dataset.
Can you provide the link of which imagenet dateset you use and how to setup the directory structure of imagenet dataset?

Thanks!

**Comment by pierlj** [comment]
Hi, 

I used ImageNet only for the computation of a regularization loss function. The main idea was to force the network to output smooth depth maps. This could be prevented by setting `mask_loss` arguments to `"none"`, it will still require some directory structure but you can let it empty. 

Concerning the directory structure, you should have something like this:
path/to/datasets/your_dataset/
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;+-- images (containing the images)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;+-- depths (containing the depth files with corresponding names as in images folder)

If you really want to use imagenet for the computation of the mask_loss you can download it and put the images under the same folder and give this path to `mask_loss_dataset` argument. You could use any other dataset for this even the same as the one used for computing regression with `mask_loss="same"`. 

Note that this may not be 100% accurate as I do not have access to the workspace I used for this project, therefore this comes only from my probably inaccurate memories.

**Comment by sp8va100** [comment]
Hi, sorry for late reply. Really appreciate for your help!

I followed your instruction and trained the network successfully with a few images in DIML indoor dataset just for testing.

And now I want to add more different dataset to train, but I am confused about the `focal` and `baseline` of the camera used for creating dataset and their unit you used.

In `train.py`, I saw that the DIML indoor dataset is labeled with `focal: 512` and `baseline: 74` and outdoor dataset is labeled with `focal: 512` and `baseline: 120`. 
However, when I read [DIML dataset's paper](https://dimlrgbd.github.io/downloads/technical_report.pdf), I only found that the indoor dataset is created by Kinect v2 with no information of focal and baseline, and the outdoor dataset is created by stereo camera with `focal: 2.8mm` and `baseline: 12cm`, so that made me confused about how to set up focal and baseline with any dataset. 

Thanks!

**Comment by pierlj** [comment]
Yes I agree it is confusing. This comes from the original implementation, I guess they used this in order to somehow normalize depth values. 

It does not really matter as long as you keep the same value for estimation and for rendering. Be aware that there are hardcoded values of the focal length inside the code so changing only the focal length in dataset params is not enough.

**Comment by sp8va100** [comment]
Understood!

By the way, can you provide the focal and baseline you use in NYUv2?
I saw you mention in other issue that you use it for training; however, I didn't see it in `training.py`.

Thanks!

**Comment by pierlj** [comment]
NYUv2 was created using a Kinect, just as for the indoor part of DIML. Therefore I assumed that I could use the same baseline. For the focal, just adapt it to the image size of NYUv2 so the ratio `focal / width` is the same. In any case, changing the focal will only change the magnitude of the depth values when the baseline is fixed.
